src/main/java/org/ibp/api/domain/location/LocationDto.java

```diff
@@ -0,0 +1,103 @@
+
+package org.ibp.api.domain.location;
+
+import org.pojomatic.Pojomatic;
+
+public class LocationDto {
+
+	private Integer id;
+	private String name;
+	private String abbreviation;
+	private Double latitude;
+	private Double longitude;
+	private Double altitude;
+	private boolean defaultLocation;
```

Comment:
@cuenyad please rename to default

Reply:
@clarysabel I used defaultLocation because default is a java reserved word.